src/utils/style-sheet.ts

```diff
@@ -1,14 +1,21 @@
+/* tslint:disable */
+
 // This file is based off glamor's StyleSheet
-// https://github.com/threepointone/glamor/blob/v2.20.40/src/sheet.js
+// Https://github.com/threepointone/glamor/blob/v2.20.40/src/sheet.js
```

Comment:
The reason why I changed the `comment-format` rule 😛 https://github.com/segmentio/remote-service-client/blob/8907f3d72b4438f065c3d91729488055aa6e80fd/tslint.json#L5